weave

```diff
@@ -1667,7 +1667,14 @@ launch_router() {
            --label=weavevolumes $WEAVEDB_IMAGE >/dev/null
     fi
 
-    RESOLV_CONF=$(chroot ${HOST_ROOT:-/} readlink -f /etc/resolv.conf)
+    if [ -f ${HOST_ROOT:-/}/etc/resolv.conf ]; then # regular file
```

Comment:
Thanks. I copied that from someone's comment on the previous version of this PR. I will switch back to `-L`